gatsby-node.js

```diff
@@ -124,6 +125,49 @@ async function sourceOcticonData({actions, createNodeId, createContentDigest}) {
   }
 }
 
+async function sourceFigmaData({actions, createNodeId, createContentDigest}) {
+  // Save the Primer Figma data to the GraphQL store
+  const json = await fetch(
+    `https://raw.githubusercontent.com/primer/figma/main/packages/web/generated/components.json`,
```

Comment:
How does this data get updated?

Reply:
We've set up [a workflow](https://github.com/primer/figma/blob/main/.github/workflows/docgen-web.yml) that updates the JSON file daily (can be triggered on demand too). Also, we're planning to hook it up to the Figma library release process that is being worked on, so it's also triggered automatically when a new release is published.